src/core/shape_inference/include/reverse_sequence_shape_inference.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+
+#include <openvino/core/validation_util.hpp>
+#include <openvino/op/reverse_sequence.hpp>
+
+namespace ov {
+namespace op {
+namespace v0 {
+template <class T>
+void shape_infer(const ReverseSequence* op, const std::vector<T>& input_shapes, std::vector<T>& output_shapes) {
+    const auto& data_pshape = input_shapes[0];
+    const auto& data_rank = data_pshape.rank();
+    using DimType = typename std::iterator_traits<typename T::iterator>::value_type;
+    NODE_VALIDATION_CHECK(op,
+                          data_rank.is_dynamic() || data_rank.get_length() >= 2,
+                          "Data input rank should be equal or greater than 2. Got: ",
+                          data_pshape);
+
+    int64_t normalized_batch_axis = normalize_axis(op, op->m_batch_axis, data_rank);
```

Comment:
Looks like we can use m_normalized_batch_axis here

Reply:
yes, fix